src/Apps/Order/Utils/shippingUtils.ts

```diff
@@ -1,16 +1,18 @@
-import { Address, emptyAddress } from "Components/Address/AddressForm"
+import { emptyAddress } from "Components/Address/AddressForm"
 import { Shipping_me$data } from "__generated__/Shipping_me.graphql"
 import { Shipping_order$data } from "__generated__/Shipping_order.graphql"
-import { pick, omit, compact } from "lodash"
+import { pick, omit, compact, isNil, omitBy } from "lodash"
 import {
   UpdateUserAddressMutation$data,
   UserAddressAttributes,
 } from "__generated__/UpdateUserAddressMutation.graphql"
 import { NEW_ADDRESS } from "Apps/Order/Components/SavedAddresses"
-import {
-  CommerceOrderFulfillmentTypeEnum,
-  SetShippingMutation$data,
-} from "__generated__/SetShippingMutation.graphql"
+import { SetShippingMutation$data } from "__generated__/SetShippingMutation.graphql"
```

Comment:
There are probably some helpers in here no longer used, and some shared logic in the other files that could be bubbled up to helpers. Not only specific to this file, there are also some overlapping types that could probably be consolidated, mostly around varying address shapes.